util/artifact/artifact.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
```

Comment:
Can you note the purpose of this file (i.e., it's working around an intentional genquery() limitation: https://docs.bazel.build/versions/main/be/general.html#genquery).